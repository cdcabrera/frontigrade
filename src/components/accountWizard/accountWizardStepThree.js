import React from 'react';

const AccountWizardStepThree = () => <div>Lorem ipsum step three.</div>;

AccountWizardStepThree.propTypes = {};

export { AccountWizardStepThree as default, AccountWizardStepThree };
