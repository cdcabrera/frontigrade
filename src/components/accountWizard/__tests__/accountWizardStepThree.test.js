import React from 'react';
import { mount } from 'enzyme';
import { AccountWizardStepThree } from '../accountWizardStepThree';

describe('AccountWizardStepThree Component', () => {
  it('should render', () => {
    const props = {};
    const component = mount(<AccountWizardStepThree {...props} />);

    expect(component.render()).toMatchSnapshot();
  });
});
