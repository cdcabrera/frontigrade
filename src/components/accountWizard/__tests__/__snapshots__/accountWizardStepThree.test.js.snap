// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AccountWizardStepThree Component should render 1`] = `
<div>
  Lorem ipsum step three.
</div>
`;
